gtfs/api/routes.py

```diff
@@ -0,0 +1,40 @@
+from rest_framework import serializers, viewsets, filters
+from gtfs.models import Route, Stop
+from gtfs.api.stops import StopSerializer
+
+
+class RouteSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Route
+        fields = ("id", "name", "stops")
+
+    name = serializers.SerializerMethodField()
+    stops = serializers.SerializerMethodField()
+
+    def get_name(self, obj):
+        return obj.short_name
+
+    def get_stops(self, obj):
+        if "stops" not in self.context:
+            self.context["stops"] = Stop.objects.all()
```

Comment:
Not really sure if this was the correct place to put this 🤔  Anyway felt like a good idea so we don't query the database for every route